### PR TITLE
[tflchef]Add is_variable on tflchef operand

### PR DESCRIPTION
**Pull request by struss**
For #4396 

This commit add `is_variable` option on recipe for tflchef.

ONE-DCO-1.0-Signed-off-by: KiDeuk Bang <rrstrous@nate.com>

### CODE REVIEW

File: compiler/tflchef/proto/tflchef.proto [hunk 1]
```diff
@@ -458,7 +459,7 @@ message SegmentSumOptions {
 }
 
 message UnidirectionalSequenceLSTMOptions {
-  optional Activation activation = 1 [default = NONE];
+  optional Activation activation = 1 [default = TANH];
```
Comment (seanshpark) [inline compiler/tflchef/proto/tflchef.proto:462]: this doesn't seem to be related with `variable`

Comment (struss) [reply compiler/tflchef/proto/tflchef.proto:462]: Oh. that's my mistake. I will fix it.

**Review by mhs4670go — APPROVED** [review]
LGTM

**Review by seanshpark — APPROVED** [review]
LGTM
